src/main/resources/schema/provider.definition.schema.v1.json

```diff
@@ -85,6 +85,26 @@
                             "type": "boolean",
                             "default": true
                         },
+                        "nullEqualsEmpty": {
+                            "description": "When set to false, this flag indicates that null and empty list are not equal, and changing the values from null to empty list would indicate diff",
+                            "type": "boolean",
+                            "default": false
+                        },
+                        "caseSensitive": {
+                            "description": "When set to true, this flag indicates that case of the string will be honored, and changing the case of the string would indicate diff",
+                            "type": "boolean",
+                            "default": true
+                        },
+                        "optionalSuffix": {
+                            "description": "When set, this flag indicates that the read handler will return this property with suffixed string",
```

Comment:
flag isn't the correct terminology here imo. Flag usually indicates a boolean value whereas this is just a string

Reply:
would metadata work better than flag?